projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionResultManager.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.service.execution;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import com.google.gson.Gson;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+
+import com.vmware.taurus.service.KubernetesService;
+import com.vmware.taurus.service.model.ExecutionResult;
+import com.vmware.taurus.service.model.ExecutionStatus;
+import com.vmware.taurus.service.model.ExecutionTerminationStatus;
+
+@Slf4j
+public class JobExecutionResultManager {
```

Comment:
Add Javadoc. What is the purpose of the class? How is it used? And so on.

Reply:
Fix